dp/drivers/i40e.c

```diff
@@ -158,7 +158,7 @@ static void dev_infos_get(struct ix_rte_eth_dev *dev, struct ix_rte_eth_dev_info
 	struct rte_eth_dev_info dpdk_dev_info;
 
 	rte_eth_dev_info_get(dev->port, &dpdk_dev_info);
-	dev_info->nb_rx_fgs = dpdk_dev_info.reta_size;
```

Comment:
This returns 512.
It seems it causes the following function to overflow and zero out fields of the queue (drv_rx_queue is null after it).
https://github.com/aroulin/ix/blob/i40e-driver-release/dp/core/ethdev.c#L260

Need to debug that. With 128 it works fine.